homework/index.js

```diff
@@ -16,27 +53,79 @@
     xhr.send();
   }
 
-  function createAndAppend(name, parent, options = {}) {
-    const elem = document.createElement(name);
-    parent.appendChild(elem);
-    Object.keys(options).forEach(key => {
-      const value = options[key];
-      if (key === 'text') {
-        elem.textContent = value;
+  function fetchRepositoryContributors(url, parent) {
+    fetchJSON(url, (err, data) => {
+      if (err) {
+        createAndAppend('div', parent, { text: err.message, class: 'alert-error' });
       } else {
-        elem.setAttribute(key, value);
+        buildRepositoryContributors(data, parent);
       }
     });
-    return elem;
+  }
+
+  function buildRepositoryInfo(RepositoryElement, parent) {
+    const repository = createAndAppend('div', parent, { id: 'repository' });
+    const description = createAndAppend('div', parent, { id: 'description' });
+    const fork = createAndAppend('div', parent, { id: 'fork' });
+    const dateRow = createAndAppend('div', parent, { id: 'date' });
+
+    createAndAppend('label', repository, { text: 'Repository Name: ' });
+    createAndAppend('a', repository, {
+      href: RepositoryElement.html_url,
+      text: RepositoryElement.name,
+      target: '_blank',
+    });
+    createAndAppend('label', description, { text: 'Description: ' });
+    createAndAppend('span', description, { text: RepositoryElement.description });
+    createAndAppend('label', fork, { text: 'Fork: ' });
+    createAndAppend('span', fork, { text: RepositoryElement.forks });
+    createAndAppend('label', dateRow, { text: 'Date: ' });
+    createAndAppend('span', dateRow, { text: RepositoryElement.updated_at });
+  }
+
+  function buildRepositoryList(repositories, parent) {
+    const top = createAndAppend('div', parent, { id: 'top' });
+    const body = createAndAppend('div', parent, { id: 'bodyTag' });
+    const rightSide = createAndAppend('div', body, { id: 'right' });
+    createAndAppend('label', top, { text: 'Select a Repository: ' });
+    const selectRepositoryMenu = createAndAppend('select', top, {
+      id: 'selectMenu',
```

Comment:
Please try to use `kebab-casing` rather than `camelCasing` when trying to enter values for HTML element's class and ID attributes. Refer the `alert-error` class for example 🙂